modules/gdscript/gdscript_editor.cpp

```diff
@@ -427,7 +428,30 @@ void GDScriptLanguage::debug_get_globals(List<String> *p_globals, List<Variant>
 }
 
 String GDScriptLanguage::debug_parse_stack_level_expression(int p_level, const String &p_expression, int p_max_subitems, int p_max_depth) {
-	return "";
+    List<String> names;
+	List<Variant> values;
+	debug_get_stack_level_locals(p_level, &names, &values, p_max_subitems, p_max_depth);
+
+	Vector<String> name_vector;
+	for (auto it = names.begin(); it != names.end(); ++it) {
+		name_vector.push_back(*it);
+	}
+
+	Array value_array;
+	for (auto it = values.begin(); it != values.end(); ++it) {
+		value_array.push_back(*it);
+	}
```

Comment:
We ban the usage of the `auto` keyword in the Godot codebase (with a few exceptions in complex lambdas) as we prefer the types to be explicit.